core/src/main/java/org/geysermc/geyser/registry/populator/Conversion685_671.java

```diff
@@ -45,6 +45,8 @@ public class Conversion685_671 {
     private static final List<Item> NEW_MUSIC_DISCS = List.of(Items.MUSIC_DISC_CREATOR, Items.MUSIC_DISC_CREATOR_MUSIC_BOX, Items.MUSIC_DISC_PRECIPICE);
 
     static GeyserMappingItem remapItem(Item item, GeyserMappingItem mapping) {
+        mapping = Conversion712_685.remapItem(item, mapping);
```

Comment:
should we renamed this class to `712_671`?

Reply:
We haven't really done that in the past so I don't think so.

Reply:
okay, but the methods don't map 685 to 671, they map 712 to 671. i probably should have written the first Conversion as an interface impl tbh, i'll try that out some other time